### DISCUSSION TITLE
forum-session-AI

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#129] [General]
# forum-session-AI

This session features demonstrations from two research teams discussing their AI-driven tools for accessibility, what is new and emerging, and how you can get involved.

https://accessibility2024.arxiv.org/forum-session-AI

**Comment by SBBCornell** [comment]
Welcome everyone! Please share thoughts and resources of interest to the community.

**Comment by rgonzalezp** [comment]
If you have any questions about this session, or you would like to get in touch please contact me at reg258@cornell.edu!

**Reply by rgonzalezp** [reply]
I am the panelist speaker Ricky Gonzalez, Ph.D. Candidate in Information Science at Cornell Tech.
